### PR TITLE
change task wording

### LINKED ISSUE #559: Change task 'close/d' wording to 'approve/d' and 'unsubmit/ted' to 'request changes'
* [x] menu items
* [x] buttons
* [x] action stream entries

DO NOT change the task model state wording, or API endpoints

### PR DESCRIPTION
**Pull request by goose-life**
closes #559

### CODE REVIEW

File: indigo_app/templates/indigo_api/_task_cards.html [hunk 1]
```diff
@@ -34,17 +34,17 @@ <h6 class="text-center mb-4">
                       <button type="submit"
                               class="dropdown-item"
                               formaction="{% url 'unsubmit_task' place=place.place_code pk=task.pk %}?next={{ request.get_full_path|urlencode }}"
-                              data-confirm="Are you sure you want to unsubmit this task?">
-                        Unsubmit
+                              data-confirm="Are you sure you want to request changes on this task?">
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/_task_cards.html]:
```suggestion
                              data-confirm="Are you sure you want to request changes to this task?">
```

File: indigo_app/templates/indigo_api/_task_cards.html [hunk 2]
```diff
@@ -34,17 +34,17 @@ <h6 class="text-center mb-4">
...
                       {% endif %}
 
                       {% if task.state == 'pending_review' and task.close_task_permission %}
                       <button type="submit"
                               class="dropdown-item"
                               formaction="{% url 'close_task' place=place.place_code pk=task.pk %}?next={{ request.get_full_path|urlencode }}"
-                              data-confirm="Are you sure you want to close this task?">
-                        Close
+                              data-confirm="Are you sure you want to approve this task?">
+                        All done
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/_task_cards.html]:
```suggestion
                        Approve
```

File: indigo_app/templates/indigo_api/task_detail.html [hunk 3]
```diff
@@ -137,11 +137,11 @@ <h5>{{ task.title }}</h5>
             {% endif %}
 
             {% if task.state == 'pending_review' and task.unsubmit_task_permission %}
-              <button class="btn btn-primary" type="submit" data-confirm="Are you sure you want to unsubmit this task?" formaction="{% url 'unsubmit_task' place=place.place_code pk=task.pk %}">Unsubmit</button>
+              <button class="btn btn-primary" type="submit" data-confirm="Are you sure you want to request changes on this task?" formaction="{% url 'unsubmit_task' place=place.place_code pk=task.pk %}">Request changes</button>
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/task_detail.html]:
```suggestion
              <button class="btn btn-primary" type="submit" data-confirm="Are you sure you want to request changes to this task?" formaction="{% url 'unsubmit_task' place=place.place_code pk=task.pk %}">Request changes</button>
```

File: indigo_app/templates/indigo_api/task_detail.html [hunk 4]
```diff
@@ -137,11 +137,11 @@ <h5>{{ task.title }}</h5>
...
             {% endif %}
 
             {% if task.state == 'pending_review' and task.close_task_permission %}
-              <button class="btn btn-success" type="submit" data-confirm="Are you sure you want to close this task?" formaction="{% url 'close_task' place=place.place_code pk=task.pk %}">All done</button>
+              <button class="btn btn-success" type="submit" data-confirm="Are you sure you want to approve this task?" formaction="{% url 'close_task' place=place.place_code pk=task.pk %}">All done</button>
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/task_detail.html]:
```suggestion
              <button class="btn btn-success" type="submit" data-confirm="Are you sure you want to approve this task?" formaction="{% url 'close_task' place=place.place_code pk=task.pk %}">Approve</button>
```

File: indigo_app/views/tasks.py [hunk 5]
```diff
@@ -249,8 +249,8 @@ def post(self, request, *args, **kwargs):
             'submit': 'submitted',
             'cancel': 'cancelled',
             'reopen': 'reopened',
-            'unsubmit': 'unsubmitted',
-            'close': 'closed',
+            'unsubmit': 'requested changes on',
```
Comment (longhotsummer) [inline indigo_app/views/tasks.py]:
```suggestion
            'unsubmit': 'requested changes to',
```

File: indigo_app/views/tasks.py [hunk 6]
```diff
@@ -259,31 +259,24 @@ def post(self, request, *args, **kwargs):
...
-                                    place_code=task.place.place_code)
-                        messages.success(request, u"Task '%s' has been unsubmitted and reassigned" % task.title)
-
-                else:
-                    if verb == 'closed' or verb == 'cancelled':
-                        task.assigned_to = None
-                    action.send(user, verb=verb, action_object=task,
-                                place_code=task.place.place_code)
-                    messages.success(request, u"Task '%s' has been %s" % (task.title, verb))
+                if change == 'unsubmit' and task.last_assigned_to:
```
Comment (longhotsummer) [inline indigo_app/views/tasks.py]: Are you happy that the special case messaging is dropped?

Comment (goose-life) [reply indigo_app/views/tasks.py]: yeah I think it makes more sense as is with the new wording – 'requested changes' implies reassigning?

Comment (goose-life) [reply indigo_app/views/tasks.py]: or are you talking about when the original assignee unsubmits the task? 
that's probably less likely now that the button reads 'request changes'?

Comment (longhotsummer) [reply indigo_app/views/tasks.py]: Yes, I was just confirming you've thought through this, because I haven't :)

**Pull request merged**